public/controllers/overview/overview.js

```diff
@@ -283,7 +283,7 @@ export class OverviewController {
         timefilter.setRefreshInterval(this.commonData.getRefreshInterval());
       }
 
-      if (typeof this.agentsCount === 'undefined') {
+      if (typeof this.agentsCount === 'undefined' && newTab === 'welcome') {
```

Comment:
**question**: I don't know what is the reason for this change. Could you elaborate?

Reply:
The request is restricted to the welcome view, since it is the only view where these values are needed.